tests/release/pipelines/e2e-fbc.go

```diff
@@ -0,0 +1,177 @@
+	var managed_fw *framework.Framework
+	var component *appservice.Component
+	var releaseCR *releaseApi.Release
+	var buildPr *v1beta1.PipelineRun
+	var releasePr *v1beta1.PipelineRun
+	var snapshot *appservice.Snapshot
+
+	stageOptions := utils.Options{
+		ToolchainApiUrl: os.Getenv(constants.TOOLCHAIN_API_URL_ENV),
+		KeycloakUrl:     os.Getenv(constants.KEYLOAK_URL_ENV),
+		OfflineToken:    os.Getenv(constants.OFFLINE_TOKEN_ENV),
+	}
+
+	BeforeAll(func() {
+
+		dev_fw, err = framework.NewFrameworkWithTimeout(
+			devWorkspace,
+			time.Minute*60,
+			stageOptions,
+		)
+		Expect(err).NotTo(HaveOccurred())
+
+		managed_fw, err = framework.NewFrameworkWithTimeout(
+			managedWorkspace,
+			time.Minute*60,
+			stageOptions,
+		)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = dev_fw.AsKubeDeveloper.HasController.CreateApplication(fbcApplicationName, dev_fw.UserNamespace)
+		Expect(err).NotTo(HaveOccurred())
+
+		componentObj := appservice.ComponentSpec{
+			ComponentName: fbcComponentName,
+			Application:   fbcApplicationName,
+			Source: appservice.ComponentSource{
+				ComponentSourceUnion: appservice.ComponentSourceUnion{
+					GitSource: &appservice.GitSource{
+						URL: fbcSourceGitUrl,
+					},
+				},
+			},
+			TargetPort: 50051,
+		}
+		component, err = dev_fw.AsKubeDeveloper.HasController.CreateComponent(componentObj, dev_fw.UserNamespace, "", "", fbcApplicationName, false, map[string]string{})
+		GinkgoWriter.Println("component : ", component.Name)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		_, err = managed_fw.AsKubeDeveloper.ReleaseController.CreateReleaseStrategy(fbcReleaseStrategyName, managedNamespace, "fbc-release", "quay.io/hacbs-release/pipeline-fbc-release:main", fbcEnterpriseContractPolicyName, fbcServiceAccountName, []releaseApi.Params{
+			{Name: "fromIndex", Value: constants.FromIndex},
+			{Name: "targetIndex", Value: constants.TargetIndex},
+			{Name: "binaryImage", Value: constants.BinaryImage},
+			{Name: "requestUpdateTimeout", Value: "420"},
+			{Name: "buildTimeoutSeconds", Value: "480"},
+		})
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = dev_fw.AsKubeDeveloper.ReleaseController.CreateReleasePlan(fbcReleasePlanName, devNamespace, fbcApplicationName, managedNamespace, "true")
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = managed_fw.AsKubeDeveloper.ReleaseController.CreateReleasePlanAdmission(fbcReleasePlanAdmissionName, devNamespace, fbcApplicationName, managedNamespace, "", "", fbcReleaseStrategyName)
+		Expect(err).NotTo(HaveOccurred())
+
+		defaultEcPolicySpec := ecp.EnterpriseContractPolicySpec{
+			Description: "Red Hat's enterprise requirements",
+			PublicKey:   "k8s://openshift-pipelines/public-key",
+			Sources: []ecp.Source{{
+				Name:   "Default",
+				Policy: []string{"github.com/enterprise-contract/ec-policies//policy/lib", "github.com/enterprise-contract/ec-policies//policy/release"},
+				Data:   []string{"github.com/enterprise-contract/ec-policies//data"},
```

Comment:
@kasemAlem , this data source is deprecated and no longer exists. Please use these instead:
```go
Data: []string{
  "oci::quay.io/redhat-appstudio-tekton-catalog/data-acceptable-bundles:latest',
  "github.com/release-engineering/rhtap-ec-policy//data"
}
```

A better approach would be to just use one of the [in-cluster resources](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/enterprise-contract/ecp.yaml) instead. These can be used by reference, e.g. `enterprise-contract-service/default`. The EC team maintains these resources and ensure updates go through testing first.

I was going to submit a PR to propose the change above but I can't tell exactly how the resource being created by the test is being used, e.g. nothing references `"fbc-pipelines-policy"` nor the constant `fbcEnterpriseContractPolicyName`.  Maybe it was disabled [here](https://github.com/redhat-appstudio/e2e-tests/commit/81fa6d364804c9d2a219cff5b07c6088a7b51efc#diff-2645a7c7a497e68fd45a5d9bb422d657d669bcdf191b2bc233ac17b77b77cc3fR94)?

Reply:
FYI @jinqi7 @cuipinghuo

Reply:
filed https://issues.redhat.com/browse/RHTAPREL-745 to track.